HW0/P5/P5.txt

```diff
@@ -1 +1,21 @@
 # Your answers here.
+
+Problem 5
+   1. 128+64+32+16+8+4+2+1 = 255s
+   2. of course it will be faster
+      --> 32+4+1 = 37s ==> 6.89x faster
+   3. 8s ==> 31.875x faster
+   4. P5.py 
+   5. need to take one second to move numbers everytime after additions are done
+      8 + 7 = 15s
+   6. For this case, performance is limited by how many bags can be moved at
+      once, like how computing power is limited by memory bandwidth.
```

Comment:
Thanks! I am studying computer architecture, so when I understood what the problem is there, I was reminded of memory bandwidth problem.